sdk/src/main/java/com/qonversion/android/sdk/internal/QonversionInternal.kt

```diff
@@ -55,6 +66,24 @@ internal class QonversionInternal(
         TODO("Not yet implemented")
     }
 
+    override suspend fun getUserInfo(): User {
+        return userController.getUser()
+    }
+
+    override fun getUserInfo(
+        onSuccess: (user: User) -> Unit,
+        onError: (exception: QonversionException) -> Unit
+    ) {
+        scope.launch {
+            try {
+                val user = userController.getUser()
```

Comment:
I think the more clear way is to call the `getUserInfo` method instead of `UserController.getUser` as there could be some additional logic for example and this method is just an extension above the suspend function.